tests/integration/Owncloud/OcisPhpSdk/NotificationTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace integration\Owncloud\OcisPhpSdk;
+
+use Owncloud\OcisPhpSdk\DriveOrder;
+use Owncloud\OcisPhpSdk\DriveType;
+use Owncloud\OcisPhpSdk\Notification;
+use Owncloud\OcisPhpSdk\Ocis;
+use Owncloud\OcisPhpSdk\OcisResource;
+use Owncloud\OcisPhpSdk\OrderDirection;
+use Owncloud\OcisPhpSdk\SharingRole;
+use Owncloud\OcisPhpSdk\User;
+
+require_once __DIR__ . '/OcisPhpSdkTestCase.php';
+
+
+class NotificationTest extends OcisPhpSdkTestCase
+{
+    private User $einstein;
+    private SharingRole $viewerRole;
+    private OcisResource $fileToShare;
+    private Ocis $ocis;
+    private Ocis $einsteinOcis;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->einsteinOcis = $this->initUser('einstein', 'relativity');
+        $token = $this->getAccessToken('admin', 'admin');
+        $this->ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
```

Comment:
you can use `$this->getOcis()` from https://github.com/owncloud/ocis-php-sdk/blob/9e6f94afdbefb224e3c919064943d77045bf139b/tests/integration/Owncloud/OcisPhpSdk/OcisPhpSdkTestCase.php#L116